kuma/users/tests/test_views.py

```diff
@@ -1307,12 +1275,29 @@ def test_signup_username_and_email_errors_event_tracking(self):
                         mock.call(
                             CATEGORY_SIGNUP_FLOW, ACTION_PROFILE_EDIT_ERROR, "username"
                         ),
-                        mock.call(
```

Comment:
As of this, your PR from last week is now going to be compliant with the new way of things. So, no need to reopen that issue/PR.